chronosphere/tfschema/log_ingest_config.go

```diff
@@ -126,3 +130,82 @@ var KeyValueLogParserSchema = &schema.Schema{
 	Optional: true,
 	MaxItems: 1,
 }
+
+var fieldNormalizationResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"custom_field_normalization": {
+			Type:     schema.TypeList,
+			Elem:     NamedStringNormalizationResource,
+			Optional: true,
+		},
+		"message": LogIngestConfigStringNormalizationSchema,
+		"primary_key": {
+			Type:     schema.TypeList,
+			Elem:     NamedStringNormalizationResource,
+			Optional: true,
+			MaxItems: 1,
+		},
+		"severity": LogIngestConfigStringNormalizationSchema,
+		"timestamp": {
+			Type:     schema.TypeList,
+			Elem:     timestampNormalizationResource,
+			Optional: true,
+			MaxItems: 1,
+		},
+	},
+}
+
+var timestampNormalizationResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"source": {
+			Type:        schema.TypeList,
+			Elem:        LogFieldPathResource,
+			Optional:    true,
+			Description: "List of field paths to check for timestamp values, in priority order. Common fields include \"timestamp\", \"@timestamp\", \"time\", \"datetime\".",
```

Comment:
No need to maintain descriptions in the tfschema -- these can easily fall out of sync w/ the API, so it's usually better to just omit them and rely exclusively on API docs.